docs/tutorials/PipelinesApi101.md

```diff
@@ -187,9 +188,12 @@ backend {
           // Pipelines and manipulate auth JSONs.
           auth = "application-default"
           // Endpoint for APIs, no reason to change this unless directed by Google.
-          endpoint-url = "https://genomics.googleapis.com/"
+          endpoint-url = "https://lifesciences.googleapis.com/"
           // This allows you to use an alternative service account to launch jobs, by default uses default service account
           compute-service-account = "default"
+		   
+          // Currently Cloud Life Sciences API is available only in `us-central1` and `europe-west2` locations.
+          location = "us-central1"	
```

Comment:
I tried europe-west2, and it actually failed with _Request contains an invalid argument_ (which is what I had seen with us-east1 as well). According to https://cloud.google.com/life-sciences/docs/concepts/locations, several different regions should work, but that is not what we are seeing. I am asking around and will update the comment as necessary.

Reply:
BT-262 was pointed out to me, so I have updated the comments to explain the situation. And I confirmed that the combination of
```endpoint-url = "https://europe-west2-lifesciences.googleapis.com/"```
and
```location = "europe-west2"```
works